code/src/AddDataToList.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from "react";
+import List from "List";
+import Create from "Create";
+import Loading from "Loading";
+
+const API = 'https://happy-thoughts-technigo.herokuapp.com/thoughts';
+const API_LIKE = (thoughtId) =>
+	`https://happy-thoughts-technigo.herokuapp.com/thoughts/${thoughtId}/like`;
+
+
+
+const AddDataToList = () => {
+	const [posts, setPosts] = useState([]);  
+	const [message, setMessage] = useState(''); 
+	const [loading, setLoading] = useState(false);
+
+
+
+/* Fetch data */
+
+	useEffect(() => {
+		Fetch();
+	}, []);
+
+	const Fetch = () => {
+		setLoading(true);
+		fetch(API)
+			.then((res) => res.json())
+			.then((data) => setPosts(data))
+			.finally(() => setLoading(false));
+	};
```

Comment:
Naming function with uppercase letter is not a correct way, only components should have uppercase letters. So instead writing Fetch(), it should be fetch()